### DISCUSSION TITLE
Any way to change the body background color without overriding a theme color?

### DISCUSSION THREAD

**Discussion by dlwiest** [mantinedev/mantine#4766] [Q&A]
By default, the background color is set to white for the light theme and a very dark gray for the dark theme. I would like to change these to a very light gray and a dark gray, respectively, but I'm not sure where the best place to set these values would be (I've been programmatically setting theme in the document on mount or when the theme changes, but that's obviously not ideal).

**Comment by rtivital** [comment]
You can add global styles to change body background-color
